shared/ui/core/src/main/java/com/ivy/ui/FormatMoneyUseCase.kt

```diff
@@ -18,12 +23,27 @@ class FormatMoneyUseCase @Inject constructor(
     private val withoutDecimalFormatter = DecimalFormat("###,###", DecimalFormatSymbols(locale))
     private val withDecimalFormatter = DecimalFormat("###,###.00", DecimalFormatSymbols(locale))
 
-    suspend fun format(value: Double): String {
-        val showDecimalPoint = features.showDecimalNumber.isEnabled(context)
+    suspend fun format(value: Double, shortenAmount: Boolean): String {
+        when (abs(value) >= THOUSAND && shortenAmount) {
```

Comment:
replace this `when` with `if` so it can be less confusing  - "when(..)true -> else ->" is a but weird